sdk/python/feast/cli.py

```diff
@@ -121,14 +121,48 @@ def feature_set():
 
 
 @feature_set.command(name="list")
-def feature_set_list():
+@click.option(
+    "--project",
+    "-p",
+    help="Project that feature set belongs to",
+    type=click.STRING,
+    default="*",
+)
+@click.option(
+    "--name",
+    "-n",
+    help="Name that feature set belongs to, may include * to match multiple feature sets eg. driver_*",
```

Comment:
A feature set doesn't belong to a name. I think this can be rephrased.

Reply:
Would this be clearer? `Name of feature set, may include * to match multiple feature sets eg. driver_*`

Reply:
What about

`Filters feature sets by name. Wildcards may be included to match multiple feature sets...`

Reply:
Updated.